packages/component/src/Composer.js

```diff
@@ -133,12 +134,36 @@ function createFocusSendBoxLogic({ sendBoxRef }) {
   };
 }
 
-function createStyleSetLogic({ styleSet, styleOptions }) {
+function createStyleSetLogic({ styleOptions, styleSet }) {
   return {
     styleSet: styleSetToClassNames(styleSet || createStyleSet(styleOptions))
   };
 }
 
+// TODO: [P3] Take this deprecation code out when releasing on or after 2019 December 11
+function patchPropsForAvatarInitials({ botAvatarInitials, userAvatarInitials, ...props }) {
+  // This code will take out "botAvatarInitials" and "userAvatarInitials" from props
+
+  let { styleOptions } = props;
+
+  if (botAvatarInitials) {
+    styleOptions = { ...styleOptions, botAvatarInitials };
+
+    console.warn('Web Chat: "botAvatarInitials" is deprecated, please use "styleOptions.botAvatarInitials" instead.');
+  }
+
+  if (userAvatarInitials) {
+    styleOptions = { ...styleOptions, userAvatarInitials };
+
+    console.warn('Web Chat: "userAvatarInitials" is deprecated, please use "styleOptions.userAvatarInitials" instead.');
```

Comment:
```'Web Chat: "userAvatarInitials" is deprecated. Please use "styleOptions.userAvatarInitials" instead.'```

Reply:
Didn't see any difference between the code and your recommendation.

Reply:
haha end the sentence after 'deprecated.' then,'Please use...'